src/db.c

```diff
@@ -1363,6 +1312,43 @@ int dbSwapDatabases(int id1, int id2) {
     return C_OK;
 }
 
+void dbSwapAllDatabases(redisDb *dbarray1, redisDb *dbarray2) {
```

Comment:
this function is not as generic as it seems (by it's name), the implementation and behavior are quite specific for placing the newly loaded temp db at the main active one, and the old active one in the temp.
logically, that's where we change the main one....
so i think we need a better name for it, and we certainly need a beefier doc comment.

Reply:
This is now swapMainDbWithTempDb